openapi/services/biblib.yaml

```diff
@@ -554,8 +554,35 @@ biblib-libraries-library_id:
         in: path
         schema:
           type: string
+      - name: start
+        description: starting entry
+        in: query
+        schema:
+          type: int
+          default: 0
+      - name: rows
+        description: number of entries to return
+        in: query
+        schema:
+          type: int
+          default: 20
+      - name: sort
+        description: entry sort order
+        in: query
+        schema:
+          type: string
+          default: 'date desc'
+
+      - name: fl
+        description: returned fields
+        in: query
+        schema:
+          type: string
+          default: 'bibcode'
+
     description: >
       View metadata and contents of a specific library.
+      Takes pagination and field arguments consistent with the "search" endpoint.
```

Comment:
Add something to this bit to clarify that the params are specifically for the records within a library. Maybe something like `...to support sorting and paging entries within a library"